tests/epre_perfect_datapackage/2018-19/epre/datapackage.json

```
@@ -10,7 +10,7 @@
             "format": "csv",
             "mediatype": "text/csv",
             "encoding": "utf-8",
-            "schema": "https://raw.githubusercontent.com/vulekamali/data-checks/master/schema/epre.json"
+            "schema": "https://raw.githubusercontent.com/fluenty/data-checks/delena-2017-18-epre/schema/epre.json"
```

revert once everything passes in CI
```suggestion
            "schema": "https://raw.githubusercontent.com/vulekamali/data-checks/master/schema/epre.json"
```